app/views/giveaways/_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if @giveaway.errors.any? %>
+  <% @giveaway.errors.full_messages.each do |msg|%>
+  <div>
+    <li><%= msg %></li>
+  </div>
```

Comment:
You have list tag without its parent for ordered/unordered list wrapped in a div container. Use either list or div container for displaying the messages unless you plan on specific custom styling

Reply:
Thank you for this, I substituted the div element with the parent <ol> .